### DISCUSSION TITLE
Is it worth providing an "official" `Dracula` style?

### DISCUSSION THREAD

**Discussion by dbitouze** [pygments/pygments#2584] [Q&A]
The [`Pygments`' `dracula` style](https://github.com/pygments/pygments/blob/master/pygments/styles/dracula.py) doesn't follow the [official `Dracula` style specifications](https://spec.draculatheme.com/#). For instance:

- [Error](https://github.com/pygments/pygments/blob/master/pygments/styles/dracula.py#L60) should be [red](https://spec.draculatheme.com/#Error) and not `foreground` (`#f8f8f2`),
- [Generic.Deleted](https://github.com/pygments/pygments/blob/master/pygments/styles/dracula.py#L52), corresponding to `Dracula`'s `DiffDeleted`, should be [red](https://spec.draculatheme.com/#DiffDeleted) and not `#8b080b`.

Is it worth providing an "official" `Dracula` style?

**Comment by birkenfeld** [comment]
Why not adapt the style to the spec instead?

**Comment by dbitouze** [comment]
> Why not adapt the style to the spec instead?

In order to not break existing documents.

**Comment by birkenfeld** [comment]
Well, either we fix it (the "breakage" should be minimal) or we keep it as is. But I definitely don't want to add another style that only varies very slightly.

**Comment by jeanas** [comment]
Agreed. We normally don't consider changing a style a "breaking change" if the look and feel stays about the same. (For example, there have been a number of changes to improve contrast, for better accessibility to people with impaired vision.)
